parm/config/gefs/config.resources

```diff
@@ -53,38 +53,38 @@ export npe_node_max
 case ${step} in
 
   "stage_ic")
-    export wtime_stage_ic="00:15:00"
-    export npe_stage_ic=1
-    export npe_node_stage_ic=1
-    export nth_stage_ic=1
+    export wtime="00:15:00"
+    export npe=1
+    export npe_node=1
+    export nth=1
     export is_exclusive=True
     ;;
 
   "waveinit")
-    export wtime_waveinit="00:10:00"
-    export npe_waveinit=12
-    export nth_waveinit=1
-    export npe_node_waveinit=$(( npe_node_max / nth_waveinit ))
-    export NTASKS=${npe_waveinit}
-    export memory_waveinit="2GB"
+    export wtime="00:10:00"
+    export npe=12
+    export nth=1
+    export npe_node=$(( npe_node_max / nth ))
+    export NTASKS=${npe}
```

Comment:
For a later PR, but what is the purpose of redefining the same thing with a different name.

Reply:
It can probably be removed in favor of `ntasks`.  There are just a few wave jobs that use it for MPMD control.